Chat/Chat.cpp

```diff
@@ -96,7 +96,7 @@ void Chat::PutMsg(const UserMsg& msg)
 void Chat::PrintSomeoneList(std::vector<std::string>& list)
 {
     chat_mutex_.Lock();
-    for (int i = 0; i < list.size(); i++)
+    for (int i = 0; i < (int)list.size(); i++)
```

Comment:
yea, just want to avoid warnings.
If i remember correctly it woun't fail even without it.